exercises/bowling/bowling_test.exs

```diff
@@ -0,0 +1,152 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("bowling.exs")
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule BowlingTest do
+  use ExUnit.Case
+
+  defp roll_from_array(game, []) do game end
+
+  defp roll_from_array(game, rolls) do
+    roll_from_array(Bowling.roll(game, hd(rolls)), tl(rolls))
```

Comment:
pattern matching for rolls here would be nice: `[h | t]` instead of `rollls` in the argument